coursework/seminar/index.md

```diff
@@ -37,57 +37,57 @@ From our past experience, students tend to ramble on and on in their presentatio
 
 In particular, each presentation is limited to 20 slides and each slide should have exactly 20 seconds of air time. This means that each presentation should be approximately 7 minutes long. Please stay within this time limit. You can choose to have a single person present or more than one but keep in mind that you have a limited time. The teaching staff WILL cut you off after 7 minutes.
 
-Perhaps it is impossible to cover all the suggested points above in 7 minutes? Well, the point here is to make a thoughtful presentation. You need to work out as a group exactly what do if you can't fit everything into 7 minutes. Sometimes less is more. Sometimes it's not. Life is full of hard questions.
+Perhaps it is impossible to cover all the suggested points above in 7 minutes? Well, the point here is to make a thoughtful presentation. You need to work out as a group exactly what to do if you can't fit everything into 7 minutes. Sometimes less is more. Sometimes it's not. Life is full of hard questions.
 
-You can present using Powerpoint, Keynote or whatever tools you wish to use. Surprise us (in a good way) if you can. BEFORE submitting please ensure that your slides auto-advance every 20 seconds. No cheating! 😜
+You can present using PowerPoint, Keynote or whatever tools you wish to use. Surprise us (in a good way) if you can. **BEFORE submitting**, please ensure that your slides auto-advance every 20 seconds. No cheating! 😜
 
 Following the presentation, there will be a short Q&A session opened to the rest of the class for a maximum of 3 minutes.
 
 ## Application Critique
 
 After listening to all the presentations, each student will be randomly assigned to write about two of the applications that were presented (but not his/her own presentation), and have to write about the following:
 
-1. Summary of what the presenting team said about the application that you think is most important. Don't just regurgitate the entire presentation. Focus on say three points and explain why you are of the view that they are the most important points. (6%)
+1. Based on what the team has shared, describe and explain what the application or technology is about. Please don't regurgitate the entire presentation. Summarise the points and keep them unopinionated. (5%)
 
-2. Was the presentation effective? Did the group show that they fully internalize the presentation techniques discussed in Lecture 3 (PUNCH->WIIFY->Plan)? (6%)
+2. Was the presentation effective? Did the group show that they fully internalised the presentation techniques discussed in Lecture 3 (PUNCH->WIIFY->Plan)? (6%)
 
-3. What are your (original) thoughts? (8%)
+3. What are your (original) thoughts? (9%)
```

Comment:
Open to feedback on whether the 1% from criteria 1 should go to criteria 2 or 3.